src/formatDate.js

```diff
@@ -50,7 +50,62 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const newDate = date;
+  let str = '';
+
+  if ((fromFormat[0] === 'YYYY') && (fromFormat[2] === 'DD')
```

Comment:
use loop and switch case, it can help you avoid 4 conditions, and make your code more clear